packages/redis-worker/src/worker.ts

```diff
@@ -513,6 +539,10 @@ class Worker<TCatalog extends WorkerCatalog> {
 
         // On success, acknowledge the item.
         await this.queue.ack(id, deduplicationKey);
+
+        if (catalogItem.cron) {
```

Comment:
We should use a try/finally block in here so we are very confident the next one gets scheduled. Otherwise until we deploy again nothing will happen